ruby/lang/security/bad-deserialization.yaml

```diff
@@ -3,10 +3,9 @@ rules:
   mode: taint
   pattern-sources:
   - pattern-either:
-    - pattern: |
-        params
-    - pattern: |
-        cookies
+    - pattern: params
+    - pattern: cookies
+    - pattern: request.env
```

Comment:
@inkz  I would say environment variables will be lower likely-hood than params, and well even cookies... so we should split it out to be another rule with a lower confidence